modular_citadel/code/modules/clothing/under.dm

```diff
@@ -0,0 +1,7 @@
+/obj/item/clothing/under/syndicate/cosmetic
```

Comment:
You might want to remove the `NO_SENSORS` flag from this variant

Reply:
Feature

Reply:
ok